lib/helpers.nix

```diff
@@ -8,13 +8,21 @@ let
   nixvimBuilders = import ./builders.nix { inherit lib pkgs; };
   nixvimTypes = import ./types.nix { inherit lib nixvimOptions; };
   nixvimUtils = import ./utils.nix { inherit lib nixvimTypes _nixvimTests; };
-  nixvimOptions = import ./options.nix { inherit lib nixvimTypes nixvimUtils; };
+  nixvimOptions = import ./options.nix {
+    inherit
+      lib
+      nixvimTypes
+      nixvimUtils
+      nixvimKeymaps
+      ;
+  };
```

Comment:
I really need to get around to refactoring this; it'd be much simpler if `helpers` itself was passed in (recursively)

Reply:
This was resolved in #1948